test/test_google_geocoder.rb

```diff
@@ -116,6 +116,23 @@ def test_google_sublocality
      end
    end
 
+   def test_google_administrative_area_level_3
```

Comment:
Assignment Branch Condition size for test_google_administrative_area_level_3 is too high. [19.24/15]<br>Inconsistent indentation detected.